src/main/java/agh/ics/oop/OptionsParser.java

```diff
@@ -0,0 +1,42 @@
+package agh.ics.oop;
+
+public class OptionsParser {
+    public MoveDirection[] parse(String[] args) {
+
+        int n = 0;
+
+        for (String arg : args) {
+            switch (arg) {
+                case "f", "l", "forward", "b", "backward", "right", "r", "left" -> n++;
```

Comment:
jest to jakieś rozwiązanie, ale fajnie byłoby dwa razy nie parsować całej listy ruchów niepotrzebnie - rozwiązań tu jest wiele, ale najbardziej pasujące tutaj: możesz zrobić na początek tablicę o rozmiarze `args.length`, a potem już po switchu skopiować pierwsze `cnt` elementów do nowej, o rozmiarze `cnt`